shutdown.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "stopping test runs..."
+for pod in `kubectl get pods | grep 'j-[a-z]-[0-9]-[a-z]' | cut -d " " -f 1`; 
+do
+  kubectl delete pod $pod;
```

Comment:
We should just add a label to the pods we spawn, 'heritage=jupyterhub-loadtest' and then do `kubectl delete pod -l heritage=jupyterhub-loadtest`.